cc/Dockerfile

```diff
@@ -1,46 +1,49 @@
 #
 # Build stage
 #
-FROM fpco/stack-build:lts as builder
+FROM haskell as builder
 MAINTAINER Pat Brisbin <pbrisbin@gmail.com>
```

Comment:
I can put my email here or leave Pat's, it looks like we both are pretty responsive so it likely doesn't matter much.